monasca-persister-python/Dockerfile

```diff
@@ -32,5 +32,6 @@ RUN /build.sh -r ${PERSISTER_REPO} -b ${PERSISTER_BRANCH} \
 
 COPY persister.conf.j2 /etc/monasca-persister/
 COPY template.py start.sh kafka_wait_for_topics.py /
+RUN pip install pykafka
```

Comment:
Please, add pykafka in line 30 so it looks like this:
`-d "influxdb Jinja2 pykafka"`